prompting/validators/reward/nsfw.py

```diff
@@ -63,15 +69,23 @@ def sum_nsfw_scores(input_ids, chunk_size):
                 return max_score
 
             # 0 when needs to be filtered out, 1 when it is safe
-            return 0.0 if sum_nsfw_scores(input_ids, chunk_size=512) > boundary else 1.0
+            nsfw_score = sum_nsfw_scores(input_ids, chunk_size=512)
+            reward_event.nsfw_score = nsfw_score
```

Comment:
why are we using `nsfw_score` instead of the default `normalized_score` that we were using before?

Reply:
so the nsfw_score would be the raw score 
nsfw_reward = nsfw_normalized would be the (0/1) after passing the boundary